### DISCUSSION TITLE
Do we have plan release the Serenity v17?

### DISCUSSION THREAD

**Discussion by goseivietnam** [primefaces/community#764] [PrimeNG Templates] [answered]
Hi,
Do we have plan release the Serenity v17?

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Unfortunately, serenity will not receive an update as it's a legacy template. However, we are preparing a "how to" guide to update templates manually. We'll add "guides" section tomorrow to the primeng.org

**Reply by milosjoka** [reply]
Hi, 
Just checked and couldn't find it, can you please send the link to the "how to" guide to update a Serenity template  manually.

**Comment by efnegrete** [comment]
Hello,
I would like to know the steps to manually update the template from Serenity to angular 17, greetings.

**Comment by quartelh** [comment]
Could you please explain why the Serenity template is now marked as ‘Legacy’? 
According to the license page both the Basic and Extended license should have ‘Free of Charge Unlimited Updates’ and ‘Lifetime Technical Support at Forum’.

The reason we bought the Serenity template is to be sure that the template would remain maintained in the future.

Can you share the 'How-to' guide to upgrade the templates manually? That way we can assess what the impact is for our solutions and make the appropriate decisions on how to proceed with the template and PrimeNG

**Reply by efnegrete** [reply]
But then why buy if they won't upgrade, about what they are not worth little, there lack seriousness.
on the other hand, it is not that the templates are not a change for using angular 17, if you look at the templates I have apart from Serenity the Ultima and is almost the same only that occupies angular 17, but nothing implementing angular 17.

**Comment by darckyn** [comment]
There is no reason and you have simply decided that serenity will no longer be supported. Lifetime support has been paid but the service is not being fulfilled.

Which theme will then be safe and which guarantees that this will not happen again?

**Comment by cagataycivici** [comment]
Hi,

Thank you for your patience. A guide is being worked on and to be released next week.

Serenity and a couple of other templates have been removed from our store due to low volume of sales.

Our team will continue to provide support at the templates forum however new versions won’t be available. A new documentation will be published soon for self maintenance as it requires “ng update” to update the Angular version, updating PrimeNG version at package.json and finally replacing the

src/assets/sass/theme/theme-base

with;

https://github.com/primefaces/primeng-sass-theme/tree/main/theme-base

We’ll also put up a video tutorial, with PrimeNG 18 theming moves into core so simply updating PrimeNG/Angular version would be enough without dealing with the sass theme. Please monitor the support ticket you have created for updates.

Regards,

Cagatay Civici
PrimeTek

**Reply by quartelh** [reply]
> Our team will continue to provide support at the templates forum however new versions won’t be available. A new documentation will be published soon for self maintenance as it requires “ng update” to update the Angular version, updating PrimeNG version at package.json and finally replacing the
> 
> src/assets/sass/theme/theme-base
> 
> with;
> 
> https://github.com/primefaces/primeng-sass-theme/tree/main/theme-base

https://github.com/primefaces/primeng-sass-theme/tree/main/theme-base has not been updated AFTER you posted this information.

WIth v18 it looks like the theming is also altered.

So what/where is the 'Our team will continue to provide support ...'  ?

**Comment by MughalOwais** [comment]
We are also using Serenity & stopped at v16 because of no further updates. We now want to upgrade. Can you please direct us to the upgrade guide where our theming & layout works as it is but we would be able to use v20. We cannot just ditch the template as we are dependent on it.
